presto-testing/src/main/java/io/prestosql/testing/TestingSystemAccessControl.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.prestosql.testing;
+
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class TestingSystemAccessControl
```

Comment:
It's neither "testing implementation of SystemAccessControl" nor "testing's utilities for SystemAccessControl" ...
I don't know what this class represents.

it may be better to just use two fields (name, properties) everywhere
otherwise call it eg `TestingSystemAccessControlConfiguration`